Dockerfile

```diff
@@ -19,8 +19,12 @@ FROM continuumio/miniconda3:4.7.12
 # Upgrade all packages to meet security criteria
 RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /workspace
+
+RUN useradd --create-home max
+WORKDIR /home/max
 RUN mkdir assets
```

Comment:
You might want to run this as `max`, otherwise you don't know what permission issues you may get into.

Also, are you sure you are running as `root`? This is what our base image says: https://hub.docker.com/r/continuumio/miniconda3

> The Miniconda distribution is installed into the /opt/conda folder and ensures that the default user has the conda command in their path.

It sounds as if the default user is not root. If the default user is indeed root, this issue can be simply resolved by running a chown command after useradd so that user `max` would have the permission to run `pip` without much trouble (`USER max` can go right before `WORKDIR`).

Reply:
Yes, we appear to currently be running as root:
```
$ docker run -it --rm quay.io/codait/max-base
(base) root@6a97020bf0e5:/workspace# whoami
root
```

Reply:
> You might want to run this as max

`sudo` is not installed by default in this container. Do you think it's worth installing `sudo` so that we can create the directory as `sudo -u mkdir .......` or should we just `mkdir` then `chown` after?

Reply:
How about moving `USER max` to right below `useradd`?

```docker
RUN useradd --create-home max
RUN chown -R max:max /opt/conda
USER max
WORKDIR /home/max
...
```

Reply:
That works. I'll push a commit.

Reply:
You might want to get `sudo` installed as many of our models need to run `apt-get`.